src/data/finsupp/basic.lean

```diff
@@ -1077,6 +1077,13 @@ finset.subset.antisymm
   (calc support f = support (- (- f)) : congr_arg support (neg_neg _).symm
      ... ⊆ support (- f) : support_map_range)
 
+lemma support_sub [add_group G] {f g: α →₀ G} :
+ support (f - g) ⊆ support f ∪ support g :=
+begin
+  rw [sub_eq_add_neg, ←@support_neg _  _ _ g],
+  convert support_add,
```

Comment:
```suggestion
  exact support_add,
```
`convert` as the last step is "`exact` but there's something nasty going on", which isn't needed here.